lib/fake_braspag/views/sales_cancel_failure.jbuilder

```diff
@@ -0,0 +1 @@
+json.array! [OpenStruct.new(Code: 114, Message: "Error")], :Code, :Message
```

Comment:
In this case nothing... is only a fake error response, the code could be anything...